pkg/apis/hub/v1alpha1/ai_service.go

```diff
@@ -50,9 +50,15 @@ type AIServiceSpec struct {
 
 // +k8s:deepcopy-gen=true
 
+type Secret struct {
```

Comment:
For the sake of consistency with other Hub resources, maybe we can call it `SecretReference`.
could you please add a description?